src/index.tsx

```diff
@@ -107,12 +108,96 @@ function Index() {
                 },
               ],
             },
+            {
+              path: 'classes/:id/homework/create/easy',
```

Comment:
Don't you think it would be convenient to have the difficulty as a param?
`path: 'classes/:id/homework/create/:difficulty`

I think it could help make this 3 giant elements into one, and make this file more readable

Reply:
Agree!